### ISSUE TITLE
Using content on :hover element breaks in Safari (iOS and desktop)

### ISSUE THREAD

**Issue by adriancervoni** [rodneyrehm/viewport-units-buggyfill#51]
I think there is no option to solve this (without changing the way buggyfill use the css content).

While the 'content' should only be used in pseudo elements, if I apply the hack on a :hover element, the animation stop working on Safari.

Here's a fiddle with the issue https://fiddle.jshell.net/nc9chuod/, is there any way to solve it?

Thanks.

**Comment by rodneyrehm** [comment]
> the animation stop working on Safari.

yeah, changing the content of generated elements tends to do that.

> is there any way to solve it?

besides not using the content hack? no, not at this time.

**Comment by adriancervoni** [comment]
Oh! ok. Maybe I could make the script search in a :before pseudo-element instead of the element itself, that way it will be more natural and separated for the rest of code.

Of course, in my scenario I'm not using any :before pseudo element in the places I need the plug.
